### PR TITLE
Scenario for adding new asset

### PR DESCRIPTION
**Pull request by kevincheng96**

### CODE REVIEW

File: scenario/utils.ts [hunk 1]
```diff
@@ -254,3 +254,8 @@ export async function fetchLogs(
     return contract.queryFilter(filter, fromBlock, toBlock);
   }
 }
+
+export function extractCalldata(data: string): string {
+  // Remove the first 4 bytes (function selector) of the transaction data
+  return '0x' + data.slice(10);
+}
```
Comment (jflatow) [inline scenario/utils.ts:261]: https://github.com/compound-finance/comet/blob/main/src/deploy/index.ts#L74-L77

**Review by jflatow — COMMENTED** [review]
💪

File: scenario/utils.ts (hunk 1 above)
Comment (kevincheng96) [reply scenario/utils.ts:261]: Oh cool, was wondering where it went. Will use this one instead

Comment (kevincheng96) [reply scenario/utils.ts]: Done

**Review by jflatow — APPROVED** [review]
LGTM

**Pull request merged**